packages/node-cli/src/CLI.js

```diff
@@ -49,18 +54,49 @@ module.exports = class CLI {
 
 			program
 				.version(CLI_VERSION, '--version')
-				.option(`${INTERACTIVE_ALIAS}, ${INTERACTIVE_OPTION}`, NodeTranslationService.getMessage(INTERACTIVE_OPTION_DESCRIPTION))
-				.on('command:*', args => {
+				.option(
+					`${INTERACTIVE_ALIAS}, ${INTERACTIVE_OPTION}`,
+					NodeTranslationService.getMessage(INTERACTIVE_OPTION_DESCRIPTION),
+					this._validateInteractive
+				)
+				.on('command:*', (args) => {
 					NodeConsoleLogger.error(NodeTranslationService.getMessage(ERRORS.COMMAND_DOES_NOT_EXIST, args[0]));
 				})
 				.usage(NodeTranslationService.getMessage(USAGE))
 				.parse(process.argv);
-
 		} catch (exception) {
 			NodeConsoleLogger.error(unwrapExceptionMessage(exception));
 		}
 	}
 
+	_validateCommandExists(commandMetadataList, commandName) {
```

Comment:
I would name the second parameter `arg2` or `secondCommandLineArg` instead of `commadName`.

Reply:
Well, I think it's more explicit to name it commandName, because we are looking if the command exists.
Of course there are the exceptions like the help or version flag.

Reply:
I don't see it that way, from my point of view we are checking if a given command line argument matches a command name, or some other options.